### ISSUE TITLE
No aria-label

### ISSUE THREAD

**Issue by judygab** [moroshko/react-autosuggest#642]
I'm getting issues when running accessibility test on auto-suggest. I read in previous issues that 'aria-label' can be passed by inputProps, but that only sets a description for the input field not the dropdown container 'react-autowhatever'. Here is an [example](https://codepen.io/Judygabeskiria/pen/KLbVNY). 

This is the exact issue that I am getting:
If the drop down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated to the input drop down menu either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.
